src/main/java/edu/ksu/canvas/attendance/services/CanvasApiWrapperService.java

```diff
@@ -42,6 +43,8 @@
 
     private EnrollmentOptionsFactory enrollmentOptionsFactory = new EnrollmentOptionsFactory();
 
+    private static final Logger LOG = Logger.getLogger(CanvasApiWrapperService.class);
```

Comment:
The sort of unwritten standard here is to declare the Logger at the top of the class. It'd be line 34 in this case. Not a big deal though.